site/content/en/docs/tasks/_index.md

```diff
@@ -44,6 +44,7 @@ As a batch user, you can learn how to:
 - [Run a Kueue managed plain Pod](run/plain_pods).
 - [Run a Kueue managed JobSet](run/jobsets).
 - [Submit jobs to MultiKueue](run/multikueue).
+- [Run an argo workflow](run/argo_workload)
```

Comment:
This is great. We also have a similar way of support for the Notebook integration: https://github.com/kubernetes-sigs/kueue/issues/3878. I think we could have a dedicated section for them like "Run workloads with Pods managed by Kueue", with this doc being the first of this kind, but Notebook a potential follow up. WDYT @kannon92 @varshaprasad96 @tenzen-y ?

Reply:
I also have #3898. So I think this is a good idea to document examples that use pod integration.

Reply:
great, I missed that, I would put them together in one sub-folder. We can maybe first merge this one, then align the other.